keps/sig-node/3573-device-plugin/README.md

```diff
@@ -575,17 +669,8 @@ protocol and are able to recover from a Kubelet crash.
 Then, as long as the Device Plugin API does not change upgrading Kubelet can be done
 seamlessly through a Kubelet restart.
 
-*Currently:*
-As mentioned in the Versioning section, we currently expect the Device Plugin's
-API version to match exactly the Kubelet's Device Plugin API version.
-Therefore if the Device Plugin API version change then you will have to change
-the Device Plugin too.
-
-*Future:*
-When the Device Plugin API becomes a stable feature, versioning should be
-backward compatible and even if Kubelet has a different Device Plugin API,
-
-it should not require a Device Plugin upgrade.
+Upgrading Kubelet can be done seamlessly through a Kubelet restart and does not
+require changes to workflow as the device plugin API is stable.
```

Comment:
is it though? Is stable in the sense that hasn't changed since 1.10, but not stable in the sense that is `v1beta1`. OTOH I think that GA'ing the device plugin API is out of scope, so we may just need to adjust the terminology here.

Reply:
Sure, the intention here was to indicate that the API hasn't changed since 1.10. I will adjust the terminology.